tasklist/els-schema/src/main/java/io/camunda/tasklist/es/RetryElasticsearchClient.java

```diff
@@ -799,4 +810,54 @@ public GetLifecyclePolicyResponse getLifeCyclePolicy(
             esClient.indexLifecycle().getLifecyclePolicy(getLifecyclePolicyRequest, requestOptions),
         null);
   }
+
+  public Map<String, IndexMapping> getIndexMappings(final String namePattern) {
+    return executeWithRetries(
+        "Get indices mappings for " + namePattern,
+        () -> {
+          try {
+            final Map<String, IndexMapping> mappingsMap = new HashMap<>();
+            final Map<String, MappingMetadata> mappings =
+                esClient
+                    .indices()
+                    .getMapping(
+                        new GetMappingsRequest().indices(namePattern), RequestOptions.DEFAULT)
+                    .mappings();
+            for (final Map.Entry<String, MappingMetadata> entry : mappings.entrySet()) {
+              final Map<String, Object> mappingMetadata =
+                  objectMapper.readValue(
+                      entry.getValue().source().string(),
+                      new TypeReference<HashMap<String, Object>>() {});
+              final Map<String, Object> properties =
+                  (Map<String, Object>) mappingMetadata.get("properties");
```

Comment:
For our current indices setup, it wont be null

but I cant affirm for the further ones,
so I will add the getOrDefault over here to avoid nullPointer exception